chains/irisnet/handlerIrisnet.go

```diff
@@ -764,6 +809,8 @@ func (h *TendermintHandler) beginServicingSpamFilter(id int) {
 	}
 }
 
+// thoroughMessageCheck is used in beginServicingSpamFilter.
+// thoroughMessageCheck verify the Messages from the Marlin Relay
```

Comment:
wrong

Reply:
Can you tell, What would be the correct Statement here!